lib/ops/opsservice/report.go

```diff
@@ -224,7 +228,9 @@ func (s *site) collectKubernetesInfo(reportWriter report.FileWriter, runner *ser
 
 	var stderr bytes.Buffer
 	err = runner.RunStream(w, &stderr, s.gravityCommand("system", "report",
-		fmt.Sprintf("--filter=%v", report.FilterKubernetes), "--compressed")...)
+		fmt.Sprintf("--filter=%v", report.FilterKubernetes),
+		"--compressed",
+		"--since", since.String())...)
```

Comment:
I wonder if it'd make sense to only append this flag if "since" is not 0. Same above.

Reply:
We can do that just to be safe.